jme3-core/src/main/java/com/jme3/material/RenderState.java

```diff
@@ -34,6 +34,8 @@
 import com.jme3.export.*;
 import com.jme3.scene.Mesh;
 import java.io.IOException;
+import java.util.Arrays;
+import java.util.Objects;

```

Comment:
Unused imports

Reply:
thanks for point out, i'll fix these with next commit